src/components/Main/ContactUs.js

```diff
@@ -0,0 +1,126 @@
+import React,{useState} from 'react';
+import axios from 'axios';
+import decoration from '../../assets/Decoration.svg';
+import FacebookIcon from '../../assets/Facebook.svg'
+import InstagramIcon from '../../assets/Instagram.svg'
+
+const ContactUs=()=>{
+    
+    const [name, setName] = useState('');
+    const [mail, setMail] = useState('');
+    const [message, setMessage] = useState('');
+    const [errname, setErrname] = useState('');
+    const [errmail, setErrmail] = useState('');
+    const [errmsg, setErrmsg] = useState('');
+    const [subsuccess, setSubsuccess] = useState(``);
+    const [wrongname,setWrongName] = useState('');
```

Comment:
If you have variable that is constructed from two word like wrongname - wrong and name it should use camelCase to show that